.gitignore

```diff
@@ -6,3 +6,4 @@ npm-debug.log
 docs
 tests/configuration
 .nyc_output
+*.patch
```

Comment:
Was using git patches to rebase branches after cascading merges. Thought that it might be helpful since this using patches is common and we would probably never include them in the repo